docusaurus/docs/Android/04-compose/03-general-customization/01-chat-theme.mdx

```diff
@@ -17,6 +17,7 @@ The `ChatTheme` component is a wrapper that **you should use as the root** of al
 * **ImageLoaderFactory**: Used to create Coil image loader instances. You can use the default image loader factory, or provide a custom one.
 * **MessageAlignmentProvider**: Used to provide an alignment for a particular message. You can use the default implementation which aligns the messages of the current user to end, or customize it according to your needs.
 * **MessageOptionsUserReactionAlignment**: Used to define how message reactions are aligned when browsing all reactions for a given message.
+* **PermissionHandlers**: Used to handle permissions inside the app without having to propagate lambdas to the call site. Default implementation of the download permission handler automatically downloads files after the permission has been granted.
```

Comment:
```suggestion
* **PermissionHandlers**: Used to handle permissions inside the app. Default implementation of the download permission handler automatically downloads files after the permission has been granted.
```

Maybe we could cut down on unnecessary jargon a bit but it's not a big deal either way.